app/goalroutes.py

```diff
@@ -0,0 +1,99 @@
+from flask import Blueprint, jsonify, abort, make_response, request
```

Comment:
Minor file naming advice: Usually when you put two words together like `goalroutes`, the Python convention is to use an underscore (so `goal_routes`, much like `make_response` in this `import` statement). It just makes it easier to read. Other languages have different ways of doing this, but that's Python's way.

Reply:
This also applies to `taskroutes`, too, obviously.